### PR TITLE
Add initial Linux install instructions

### PR DESCRIPTION
**Pull request by stuartleeks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,14 @@ scoop bucket add azbrowse https://github.com/lawrencegripper/scoop-bucket.git
 scoop install azbrowse
 ```
 
+### Linux (via Releases tar.gz)
+
+Grab the url to the `.tar.gz` for the latest release for your platform/architecture. E.g. `https://github.com/lawrencegripper/azbrowse/releases/download/v1.1.193/azbrowse_linux_amd64.tar.gz`
+
+Download the release (either via the browser or `wget https://github.com/lawrencegripper/azbrowse/releases/download/v1.1.193/azbrowse_linux_amd64.tar.gz`).
+
+Extract the binary from the archive to a suitable location (here we're using `/usr/bin` for convenience): `tar -C /usr/bin -zxvf azbrowse_linux_amd64.tar.gz azbrowse`
```
Comment (lawrencegripper) [inline README.md]: Do we need a `chmod +x`?

Comment (stuartleeks) [reply README.md]: Done in latest commit :-)

**Comment by stuartleeks** [comment]
Looking at the build output, it looks like the failure is unrelated to my docs change :-)

**Comment by lawrencegripper** [comment]
Yeah looks like `golang-ci` spellcheck function has got a bit more militant :(

**Comment by stuartleeks** [comment]
And some of what they are flagging doesn't look like an issue to me. It is picking up on 'analyses' but I think it is being used as the plural of 'analysis'.

**Comment by stuartleeks** [comment]
Let me know if you want any more changes to this PR :-)

**Comment by lawrencegripper** [comment]
Yeah pushed some config to hopefully ignore these ones. Once builds good then happy to merge :)

Thanks for the work on this one.

**Pull request merged**